src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HPack/HPackEncodingException.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0.
+// See THIRD-PARTY-NOTICES.TXT in the project root for license information.
+
+namespace System.Net.Http.HPack
+{
+    internal sealed class HPackEncodingException : Exception
+    {
+        public HPackEncodingException()
+            : base()
```

Comment:
`: base()` here is not necessary.

The exception also needs to be annotated as [Serializable] and provide the appropriate deserialization ctor.